homeassistant/components/vizio/config_flow.py

```diff
@@ -58,14 +85,13 @@ class VizioConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     @staticmethod
     @callback
-    def async_get_options_flow(config_entry):
+    def async_get_options_flow(config_entry: ConfigEntry) -> VizioOptionsConfigFlow:
         """Get the options flow for this handler."""
-        return VizioOptionsConfigFlow(config_entry)
+        return VizioOptionsConfigFlow(config_entry=config_entry)
```

Comment:
Why are we now passing positional arguments as named arguments everywhere?

Reply:
I can revert this change, thought it would be helpful if I was coming back to this in the future but to your point the variables are self describing so maybe that is unnecessary

Reply:
Yes, please revert all of those changes.